addons/binding/org.openhab.binding.amazondashbutton/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry exported="true" kind="lib" path="lib/jna-4.2.1.jar"/>
```

Comment:
This jar seems to contain the native lib for MacOS, but not for any other OSes.
You might want to either add further native libs in here or enhance the installation instructions by e.g. requiring an
```
apt-get install libjna-java
```
on a RaspPi and making the .so available to the runtime (LD_LIBRARY_PATH? `ln -s' into jre/lib/aarch32?)